robolectric-shadows/shadows-maps/src/main/java/org/robolectric/shadows/ShadowMapView.java

```diff
@@ -42,9 +44,8 @@
   private boolean preLoadWasCalled;
   private boolean canCoverCenter = true;
 
-  public ShadowMapView(MapView mapView) {
-    realMapView = mapView;
-    zoomButtonsController = new ZoomButtonsController(mapView);
+  public ShadowMapView() {
+    zoomButtonsController = new ZoomButtonsController(realMapView);
```

Comment:
You are right of course, glancing it over I mistook this for __constructor__, moved initialization there now where it is guaranteed. See https://github.com/robolectric/robolectric/blob/master/robolectric/src/test/java/org/robolectric/internal/bytecode/ShadowWranglerTest.java#L48